src/Components/ProfileMainPage.js

```diff
@@ -95,7 +95,7 @@ export default function ProfileMainPage() {
           ))}
           {!profile?.savedLists?.length && (
             <Grid item xs={12}>
-              <NoResultsImage />
+              <NoResultsImage tile="true" />
```

Comment:
nit: `tile={true}`.  the string "true" is also truthy, but not a boolean, so maybe not quite as accurate.